examples/worker/Pulumi.yaml

```diff
@@ -0,0 +1,12 @@
+name: cloudflare-worker
+description: A Hello-World implementation of a Cloudflare Worker.
+runtime: nodejs
+template:
```

Comment:
Can leave this template section off for the examples here (since there is no deploy button).

Reply:
I thought it would be useful to understand which configuration is required. I did not expect those parameters until CLI asked.